docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -96,9 +98,121 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Creating systemd unit file for a pod with `--new` flag
+
+As its seen the generated unit files with `--new` flag has ExecStartPre and ExecStopPost service actions. ExecStartPre makes sure that each time the service is started it deletes any old reference left and then spins up a new instance. ExecStopPost removes the relevant instances after services are stopped.
```

Comment:
I think that's a bit complicated. Use simpler sentences, for example:

"The `--new` flag generates systemd unit files that create and remove containers at service start and stop. Such unit files are not tied to a single machine and can easily be shared with and used on other machines."

The focus is shifted from describing the output to describing how it can be used.

Reply:
done.